### PR TITLE
Plugin Details: Spacing and Padding fixes

### LINKED ISSUE #57994: Plugin Details: Spacing and Padding fixes
### Details

Let's fix the following issues
image

image

Designs
SS 2021-11-12 at 13 14 00

related convo: p58i-brP-p2#comment-52450

### Checklist

- [ ]
- [ ]

### Related

_No response_

### PR DESCRIPTION
**Pull request by WBerredo**
#### Changes proposed in this Pull Request

Fix the spacing on the Plugin Details page.

#### Testing instructions
* From a Businesss or Ecommerce plan,  go to plugins page
* Select a plugin 
* Check the design matches with the below:|

Screen Shot 2021-11-12 at 10 07 53

---

Fixes #57994

### CODE REVIEW

File: client/my-sites/plugins/style.scss [hunk 1]
```diff
@@ -156,16 +156,22 @@ body.is-section-plugins:not( .is-nav-unification ) {
 	}
 }
 
+.plugin-details__tags {
+	position: absolute;
+	font-size: $font-body-extra-small;
+	color: var( --studio-gray-40 );
+	margin-top: 85px;
```
Comment (cpapazoglou) [inline client/my-sites/plugins/style.scss]: When setting an absolute element it's suggested that we always declare at least `top, left`

Can we set the `padding-top: 100px` to a variable and use `top: calc( $padding-top - 15px)` here? That way those two implementations would be bound together and we could avoid problems due to tags being `absolute`.

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
Looks great @WBerredo, let's just find a way to make this `absolute` position more apparent to the relative one for our future selves.

File: client/my-sites/plugins/style.scss (hunk 1 above)
Comment (WBerredo) [reply client/my-sites/plugins/style.scss]: Added the suggestion [here](https://github.com/Automattic/wp-calypso/pull/58005/commits/57d99950207e524a5d9292b19256307140c5689c).

**Review by cpapazoglou — APPROVED** [review]
LGTM!

**Pull request merged**